test/runtime/samples/key-block-static-if/main.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+	let slide = 0;
+	let num = false;
+	
+	const changeNum = () => num = !num;
+</script>
+
+<section>
+	{#key slide}
+		{#if num}
+			<div>First</div>
+		{/if}
+	{/key}
+	<div>Second</div>
+</section>
+
+<button on:click={changeNum}>Click</button>
```

Comment:
`eslint` enforces a new line at the end of files, but it looks like we have only set it up to run against `js` and `ts` files (probably because we have some `.svelte` files in the test with purposefully malformed content for testing)

Reply:
eslint-plugin-svelte3 wouldn't complain about this anyway. The linting rules just apply to the parts that are JS (the script tags and the template expressions), and it doesn't extrapolate any of them to the component itself.